content/project-news/reactos-0414-released.md

```diff
@@ -0,0 +1,112 @@
+---
+title:       "ReactOS 0.4.14 released"
+author:      "ReactOS Team"
+date:        XXXX-XX-XX
+tags:        [ "release" ]
+banner:      "null"
+---
+
+The ReactOS Team is pleased to announce the release of version 0.4.14.
+As with every other release, we're regularly noting improvements and updates to keep you in touch with what is being done to ReactOS.
+In this release, improvements range from FreeLoader fixes, Shell features, kernel fixes, NEC PC-9800 support, NetKVM VirtIO bringup and to further work to the Xbox port and miscellaneous stuff.
+
+## Send To feature and Shell improvements
+The main highlight of this release is the incremental amount of improvements done to the Shell component of ReactOS, a highly vital model responsible for the interaction with the system and the user.
+Katayama Hirofumi MZ is the pioneer of Send To implementation, a feature of the Shell which the user can send files or directories to a certain location. 
+
+In addition, "Open File Location" and "Open Command prompt here" were also implemented thanks to him.
+Apart from the aforementioned features, let's not forget the bug fixes that affected the operability of the Shell. Katayama Hirofumi MZ fixed the scroll selection and Mark Jansen improved some COM (Component Object Model) interfaces so that certain drag and drop regressions could be fixed.
```

Comment:
```suggestion
Apart from the aforementioned features, let's not forget the bug fixes that affected the operability of the Shell.
Katayama Hirofumi MZ fixed the scroll selection and Mark Jansen improved some COM (Component Object Model) interfaces so that certain drag and drop regressions could be fixed.
```